config/peerpods/podvm/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,82 @@
+# build pause
+FROM registry.k8s.io/pause:3.9 as pause-bin
+
+FROM registry.redhat.io/ubi9/ubi-minimal:9.5 as pause
+RUN curl -L https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64 -o /usr/bin/umoci && chmod +x /usr/bin/umoci
+RUN umoci init --layout pause && umoci new --image pause:k8s
+RUN umoci config --image pause:k8s --config.entrypoint=/pause --author="OSC"
+RUN umoci unpack --rootless --image pause:k8s /pause
+COPY --from=pause-bin /pause /pause/rootfs/pause
```

Comment:
@bpradipt do you have more elegant suggestion for creating the pause bundle? (better to avoid binary download)

Reply:
Which binary are you referring to ?

Reply:
I wasn't clear, I'd prefer to avoid using umoci which doesn't seems to exist in normal repos